src/components/auth-module/login-view/login-view.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import { translateOptions } from '../../../i18n/config';
+import { Button, Form } from 'semantic-ui-react';
+
+import ErrorMessageView from '../../common/error-message-view';
+import { isValidEmail } from '../../../utils/validation-utils';
+
+class LoginView extends Component {
+  /***************************
+   * CONSTRUCTOR
+   ***************************/
+  constructor(props) {
+    super(props);
+
+    this.state={
+      fields: {
+        email: '',
+        password: ''
+      },
+      serverError: "",
+      errors: {}
```

Comment:
Good standard. This should probably be captured in our frontend-guidelines repo.
Make sure this is followed throughout.

Reply:
Cool.